lib/hydra/pcdm/models/concerns/object_behavior.rb

```diff
@@ -26,13 +26,22 @@ module ObjectBehavior
 
       directly_contains :files, has_member_relation: RDFVocabularies::PCDMTerms.hasFile,
         class_name: "Hydra::PCDM::File"
-
     end
 
     module ClassMethods
       def indexer
         Hydra::PCDM::ObjectIndexer
       end
+
+      def create_reflection(macro, name, options, active_fedora)
```

Comment:
Please add a comment here about what you've done. Specify that you are overriding https://github.com/projecthydra/active_fedora/blob/36d5730be41bb597d4ff15abc49a56139485f9f3/lib/active_fedora/reflection.rb#L11